DataHub/Integration/01_ManageAttributes.md

```diff
@@ -159,19 +176,19 @@ At least one attribute has been created, see [Create an attribute](#create-an-at
 1. Select the checkbox of the attribute you want to delete in the list of attributes.   
     The editing toolbar is displayed above the attributes list.
 
-2. Click the ![Delete](../../Assets/Icons/Trash03.png "[Delete]") (Delete) button in the editing toolbar.    
+2. Click the [DELETE] button in the editing toolbar.    
     The *Confirm deletion of attribute?* window is displayed.
 
     ![Confirm deletion of attribute](../../Assets/Screenshots/DataHub/Settings/Attributes/ConfirmDeletionAttribute.png "[Confirm deletion of attribute]")
 
-    > [Info] All dependencies to attribute sets, ETL mappings and entity entries are displayed in this window. Be aware that problems may occur if you delete an attribute with existing dependencies. For defining attributes, the [CONFIRM] button is not displayed in the *Confirm deletion of attribute?* window as they cannot be deleted.   
+    All dependencies to attribute sets, ETL mappings and entity entries are displayed in this window. For defining attributes, the [CONFIRM] button is not displayed in the *Confirm deletion of attribute?* window as they cannot be deleted.   
+
+    > [Caution] Deleting will permanently remove the selected data. The deletion cannot be undone and the deleted data cannot be restored. Problems may occur due to unresolved dependencies. Make sure you really want to delete the selected data.
```

Comment:
Make sure, you really want.... Ist keine nette Ansprache.-> Motive werden in Frage gestellt
Vorschlag: Make sure that it is unavoidable to delete the selected data.

Reply:
Ich notiere es in den Verbesserungsvoschlägen. Diese Caution-Boxen haben wir überall mit dieser Wortwahl. Wir können die mal bearbeiten, aber dann würde ich es durchgängig machen.